examples/js/loaders/FBXLoader.js

```diff
@@ -1923,19 +1923,24 @@
 		addAnimations( sceneGraph, animations );
 
 
-		// Parse ambient color - if it's not set to black (default), create an ambient light
-		var ambientColor = FBXTree.GlobalSettings.properties.AmbientColor.value;
-		var r = ambientColor[ 0 ];
-		var g = ambientColor[ 1 ];
-		var b = ambientColor[ 2 ];
+    // Parse ambient color - if it's not set to black (default), create an ambient light
```

Comment:
Just some style feedback: The indentation is not correct here

Reply:
Oops, fixed 😄